src/rpdk/core/contract/suite/handler_create.py

```diff
@@ -73,3 +76,15 @@ def contract_create_list(created_resource, resource_client):
     _input_model, created_model, _request = created_resource
     assert test_model_in_list(resource_client, created_model)
     test_read_success(resource_client, created_model)
+
+
+@pytest.mark.create
+@skip_not_taggable
+def contract_create_taggable(resource_client):
+    assert (
+        resource_client.metadata_contains_tag_property()
+    ), "Resource marked taggable but tagProperty attribute missing in tagging metadata."
+    input_model = resource_client.generate_create_example()
+    assert resource_client.validate_model_contain_tags(
```

Comment:
Should we move it as part of the normal `contract_create_delete`  instead of a new contract test case?  

Also if we introduce this new, will it break some customers if they don't include tags in their contract test input data? how should we rollout this change without breaking customers?

Reply:
have added @skip_no_tagging so that if tagging metadata is missing, we will skip those new test